aws/environments/jrgm.yml

```diff
@@ -0,0 +1,26 @@
+---
+region: us-west-2
+subdomain: jrgm.dev
```

Comment:
Yeah, kinda wanted to commit this, but maybe I should drop this since it's not really part of this PR.